### DISCUSSION TITLE
HELP!! Why pictures & pdfs aren't visible suddenly...

### DISCUSSION THREAD

**Discussion by Sinhaaz** [community/community#55210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It was visible earlier but suddenly maybe I've done something.... (I've no idea related to what I've done unknowingly)
No pictures and even pdfs are visible in the repo(pictures which was visible in the readme are now not visible.

Anyone knows the reason???
And how can I fix it?
![Screenshot (30)](https://github.com/community/community/assets/105969236/28eb4a4c-6706-4a84-9aea-877f3b41e1a8)
![Screenshot (31)](https://github.com/community/community/assets/105969236/d0b0a5f2-0612-4844-a3cf-169500e64a3c)

**Comment by airtower-luna** [comment]
Can you reach `raw.githubusercontent.com`? That's where the broken files would load from. Your profile says you're in India, at least one Indian provider (Jio) has been frequently reported to block `raw.githubusercontent.com` at the DNS level (see #32889 for an example). If it's that, using an alternative DNS server should help.

**Reply by Sinhaaz** [reply]
No I can't reach "raw.githubusercontent.com".... What should I need to do now?

**Reply by airtower-luna** [reply]
As I said, the first thing I'd try is to use another DNS server. I'd use [Quad9](https://quad9.net/) (9.9.9.9, 2620:fe::fe), but there are others.

**Reply by Sinhaaz** [reply]
Can you see the pictures and pdfs in my repos and readme?

**Reply by airtower-luna** [reply]
Yep, works just fine for me.
